compiler/ir_utils.py

```diff
@@ -143,6 +143,29 @@ def format_expanded_arg_char(arg_char):
         ## TODO: Make this correct
         raise ValueError
 
+def format_arg_to_raw_chars(arg_char_list):
+    chars = [format_arg_to_raw_char(arg_char) for arg_char in arg_char_list]
+    return "".join(chars)
+
+def format_arg_to_raw_char(arg_char):
```

Comment:
I don't see the difference of this from this: `format_arg_char`.

Reply:
If there is a difference we need to be explicit and write it in a comment.

Reply:
This function calls format_arg_to_raw_char which doesn't add escape characters

Reply:
Rewrote the solution to not more functions